packages/backend/src/generated/swagger.json

```diff
@@ -715,7 +715,6 @@
                     "scatter",
                     "area",
                     "mixed",
-                    "pie",
```

Comment:
fyi: I think we are rolling back changes from main.

Reply:
let me rebase from main again